src/ResourceManager/Resources/Commands.Resources/DenyAssignments/GetAzureDenyAssignmentCommand.cs

```diff
@@ -0,0 +1,188 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+using Microsoft.Azure.Commands.Resources.Models;
+using Microsoft.Azure.Commands.Resources.Models.Authorization;
+using Microsoft.Azure.Graph.RBAC.Version1_6.ActiveDirectory;
+using Microsoft.WindowsAzure.Commands.Common;
+using System;
+using System.Collections.Generic;
+using System.Management.Automation;
+
+namespace Microsoft.Azure.Commands.Resources
+{
+    /// <summary>
+    /// Filters deny assignments
+    /// </summary>
+    [Cmdlet("Get", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "DenyAssignment", DefaultParameterSetName = ParameterSet.Empty), OutputType(typeof(PSDenyAssignment))]
+    public class GetAzureDenyAssignmentCommand : ResourcesBaseCmdlet
+    {
+        private const string DenyAssignmentIdParameterSet = "DenyAssignmentIdParameterSet";
+        private const string DenyAssignmentNameParameterSet = "DenyAssignmentNameParameterSet";
+
+        [Parameter(Mandatory = true, ValueFromPipelineByPropertyName = true, ParameterSetName = ParameterSet.ObjectId,
+            HelpMessage = "The user or group object id.")]
+        [Parameter(Mandatory = true, ValueFromPipelineByPropertyName = true, ParameterSetName = ParameterSet.ResourceGroupWithObjectId,
+            HelpMessage = "The user or group object id.")]
+        [Parameter(Mandatory = true, ValueFromPipelineByPropertyName = true, ParameterSetName = ParameterSet.ResourceWithObjectId,
+            HelpMessage = "The user or group object id.")]
+        [Parameter(Mandatory = true, ValueFromPipelineByPropertyName = true, ParameterSetName = ParameterSet.ScopeWithObjectId,
+            HelpMessage = "The user or group object id.")]
+        [Alias("PrincipalId")]
+        public Guid? ObjectId { get; set; }
```

Comment:
@kirankulkarni747 I'm not sure if it applies to deny assignments, but we have an [open issue](https://github.com/Azure/azure-powershell/issues/7736) to change instances of object id from `Guid` to `string`, so you may need to make the same change here

Reply:
We don't need to change the type from Guid to string because the above referenced issue applies to Azure Stack and deny assignments are not supported in Azure Stack.